vitest.workspace.ts

```diff
@@ -28,6 +28,12 @@ const workspaces = packagedEmbedTestsOnly
           include: ["packages/**/*.integration-test.ts", "apps/**/*.integration-test.ts"],
           exclude: ["**/node_modules/**/*", "packages/embeds/**/*"],
           setupFiles: ["setupVitest.ts"],
+          globalSetup: ["./apps/api/v1/test/setup/globalSetup.ts"],
+          poolOptions: {
+            threads: {
+              singleThread: true,
+            },
+          },
```

Comment:
to prevent any race condition https://vitest.dev/guide/cli.html#pooloptions-threads-singlethread